src/components/UpdateArticleVotes.jsx

```diff
@@ -0,0 +1,27 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+
+function UpdateArticleVotes (){
+    const article = useParams()
+    const articleChoice = article.article_id
+    let votes = 0
+    let [newVotes, setNewVotes] = useState(votes)
+    useEffect(()=>{
+        axios.patch(`https://northcoders-news-api-bjpy.onrender.com/api/articles/${articleChoice}`,{inc_votes: newVotes})
+    .then((response)=>{
+        
+    })
+},[newVotes])
+    return (
+        <>
+        <br/>
```

Comment:
<br>s shouldn't be used for styling, use CSS for this, and you should use an appropriate HTML tag as the parent tag to this rather than `<>` this will help you style